out_writeapi.go

```diff
@@ -313,6 +338,12 @@ func FLBPluginInit(plugin unsafe.Pointer) int {
 		log.Printf("Invalid Exactly_Once parameter in configuration file: %s", err)
 	}
 
+	//optional num synchronous retries parameter
```

Comment:
Please augment the comment to indicate this is used only for exactly-once.

Reply:
Done.